src/DrDotnet.Profilers/api/clr_profiler_info.rs

```diff
@@ -74,7 +74,7 @@ impl ClrProfilerInfo {
         }
     }
     
-    pub fn set_attached_status(&self, status: AttachedStatus) {
+    pub(in crate::api) fn set_attached_status(&self, status: AttachedStatus) {
```

Comment:
What does `(in crate::api)` change?

Reply:
It makes it only available to api module, and not profilers, so that we can't set the status manually in profilers